### ISSUE TITLE
PsychoPy preferences should add "Runner" as "default view" option

### ISSUE THREAD

**Issue by hoechenberger** [psychopy/psychopy#2814]
Currently, only "Coder" and "Builder" are offered:
<img width="446" alt="Screen Shot 2020-02-24 at 09 46 34" src="https://user-images.githubusercontent.com/2046265/75138499-acd1fb80-56ea-11ea-8b0b-558424dd2623.png">

**Comment by peircej** [comment]
Yes. Actually we should rethink the view menu so user can cycle through all open windows. Right now there only options are "switch to Builder" (but which window?) Or "switch to Coder"
